src/events/dns.js

```diff
@@ -0,0 +1,358 @@
+const uuid4 = require('uuid4');
+const dns = require('dns');
+const shimmer = require('shimmer');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const serverlessEvent = require('../proto/event_pb.js');
+const eventInterface = require('../event.js');
+const errorCode = require('../proto/error_code_pb.js');
+const { isBlacklistURL } = require('.././helpers/events');
+
+const URL_BLACKLIST = {
+    'tc.epsagon.com': 'endsWith',
+};
+
+// Resouece records types
+const rrtypesMethods = {
+    resolveAny: 'ANY',
+    resolve6: 'AAAA',
+    resolve4: 'A',
+    resolveCname: 'CNAME',
+    resolveMx: 'MX',
+    resolveNaptr: 'NAPTR',
+    resolveNs: 'NS',
+    resolvePtr: 'PTR',
+    resolveSoa: 'SOA',
+    resolveSrv: 'SRV',
+    resolveTxt: 'TXT',
+};
+
+/**
+ * Initial Dns event.
```

Comment:
creates and initial DNS event in the protobuf format (you just wrote the function name..)